modules/references/server/models/reference.server.model.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var mongoose = require('mongoose'),
+    uniqueValidation = require('mongoose-beautiful-unique-validation'),
+    Schema = mongoose.Schema;
+
+/**
+ * ReferenceUser Schema
+ */
+var ReferenceSchema = new Schema({
+  created: {
+    type: Date,
+    default: function () { return Date.now(); }, // Date.now is wrapped for sinon.useFakeTimers()
+    required: true
+  },
+  /*
+  modified: {
+    type: Date,
+    default: Date.now
+  },
+  */
+  public: {
+    type: Boolean,
+    default: false,
+    required: true
+  },
+  userFrom: {
+    type: Schema.ObjectId,
+    ref: 'User',
+    required: true
+  },
+  userTo: {
+    type: Schema.ObjectId,
+    ref: 'User',
+    required: true
+  },
+  met: {
+    type: Boolean,
+    default: false,
+    required: true
+  },
+  hostedMe: {
+    type: Boolean,
+    default: false,
+    required: true
+  },
+  hostedThem: {
+    type: Boolean,
+    default: false,
+    required: true
+  },
```

Comment:
I wonder if this is worth putting into a sub schema of `interaction`, so:

```js
{
  interaction: {
    met: true,
    hostedMe: false,
    hostedThem: false
  }
}
```

Advantages:
* validation error for "no interaction selected" can have a field name
* it more clearly shows the intention (and where to add new ones), given mongodb is a document store it's good at nested data

... or an array like is used elsewhere for other things?